ehrapy/api/encode/encode.py

```diff
@@ -494,6 +542,134 @@ def _add_categoricals_to_uns(ann_data: AnnData, categorical_names: List[str]) ->
             elif var_name in categorical_names:
                 ann_data.uns["original_values_categoricals"][var_name] = ann_data.X[::, idx : idx + 1]
 
+    @staticmethod
+    def _extract_autodetect_and_encoding_modes(
+        identifier: str, autodetect: Dict, encodings: Dict[str, Dict[str, List[str]]]
+    ) -> (bool, Optional[Dict]):
+        """
+        Extract the index column (if any) and the columns, for obs only (if any) from the given user input.
+        This function is only called when dealing with mimicIII datasets; when parsing multiple files.
```

Comment:
Maybe we should not mention MIMIC-III (it's not mimicIII) here, but just keep the comment general. Or emphasize that MIMIC-III is just an example of one such use-case.